force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -83,7 +112,14 @@ public with sharing class VariablesDataTypesOperators {
      * @return True if the number is even, False otherwise.
      */
     public static Boolean isEven(Integer num) {
-        return null; // Replace null with the variable you used to store the result
+        Boolean result;
+        Integer remainder = math.mod(num,2);
+        if (remainder == 1) {
+            result = false;            
+        } else {
+            result = true;
+        }
```

Comment:
Conditions line `(remainder == 1)` are already boolean, so you can do `result = (remainder == 0)`; instead of separately setting true or false.